wagtail/admin/views/pages/history.py

```diff
@@ -8,12 +11,32 @@
 from django.utils.translation import gettext as _
 
 from wagtail.admin.auth import user_has_any_page_permission, user_passes_test
-from wagtail.admin.filters import PageHistoryReportFilterSet
+from wagtail.admin.filters import DateRangePickerWidget, WagtailFilterSet
 from wagtail.admin.views.reports import ReportView
+from wagtail.core.log_actions import page_log_action_registry
 from wagtail.core.models import (
     Page, PageLogEntry, PageRevision, TaskState, UserPagePermissionsProxy, WorkflowState)
 
 
+def get_audit_log_users_queryset(request):
```

Comment:
question: this is also defined in `wagtail/admin/views/reports/audit_logging.py`. is that intentional?

Reply:
Didn't actually notice that, thanks for spotting!

This should probably be moved to a method on ``PageLogEntry``?

Reply:
That is a sensible suggestion!